# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1758995692 -7200
# Parent  ed336691c75deb6406502826daa20ab2a83c7c36
Misc: Use MRUTime for recent folders instead of MRMTime.

diff --git a/mailnews/base/content/folder-menupopup.js b/mailnews/base/content/folder-menupopup.js
--- a/mailnews/base/content/folder-menupopup.js
+++ b/mailnews/base/content/folder-menupopup.js
@@ -550,13 +550,13 @@
         }
 
         switch (specialType) {
           case "recent":
-            // Find the most recently modified ones.
+            // Find the most recently used ones.
             specialFolders = lazy.FolderUtils.getMostRecentFolders(
               specialFolders,
               Services.prefs.getIntPref("mail.folder_widget.max_recent"),
-              "MRMTime"
+              "MRUTime"
             );
             break;
           case "favorites":
             specialFolders = specialFolders.filter(folder =>
